src/veritysetup/veritysetup-generator.c

```diff
@@ -22,53 +22,64 @@
 #include "string-util.h"
 #include "unit-name.h"
 
-#define SYSTEMD_VERITYSETUP_SERVICE "systemd-veritysetup@root.service"
+#define ROOT_NAME "root"
+#define USR_NAME "usr"
```

Comment:
let's not use macros for this. Just use the literal strings. they are short, hard to mistype, and so easy to grok. no need for the extra indirection